src/main/java/me/leoko/advancedban/utils/DynamicDataSource.java

```diff
@@ -22,7 +22,10 @@ public DynamicDataSource(boolean preferMySQL) throws ClassNotFoundException {
             config.setUsername(usrName);
             config.setPassword(password);
         } else {
-            Class.forName("org.hsqldb.jdbc.JDBCDriver");
+        	// No need to worry about relocation because the maven-shade-plugin also changes strings
+        	String driverClassName = "org.hsqldb.jdbc.JDBCDriver";
```

Comment:
Used TABs instead of spaces for code indentation

Reply:
Interesting. My IDE uses tabs by default. However, I was trying to copy the style of the surrounding code, which on my machine seems to use spaces.

I'll change to tabs nonetheless.

Reply:
Note: Leoko and I realised spaces should have been used after all.